src/cache/cache.ts

```diff
@@ -246,11 +246,6 @@ export function createCache(ctx: Context, bundleContext: IBundleContext): ICache
     const meta = findModuleMeta(absPath);
 
     const metaPackage = packages[meta.packageId];
-    // not present in meta
-    if (!meta) {
```

Comment:
`meta.pacakgeId` above will have thrown already if `!meta`, so this code block isn't reachable.